### DISCUSSION TITLE
Stuck after update to v7

### DISCUSSION THREAD

**Discussion by patrickhoehn** [deployphp/deployer#2316] [Help needed]
I just updated to v7 and followed the UPGRADE.md but now I get stuck at `⠏ ick@domain.com's password:   ⠇ `. It looks like the user name I defined is not added before the @ but "ick" (whatever this is, maybe the last part of my local user name?)

**Comment by antonmedv** [comment]
Please show full deploy.php. Did you setup remote_user?

**Reply by patrickhoehn** [reply]
Thank you for the fast reply! Changing user to remote_user fixed it (maybe add this to your UPGRADE.md?). Now I'm getting "Can't detect http user name." :/

**Reply by patrickhoehn** [reply]
I set the http_user manually to "www-data" but I don't know if this really matches to the settings of my host. Is there a way to find out which http_user my host is using?

**Reply by antonmedv** [reply]
Updated docs. http_user usually it's you web server user.

**Reply by patrickhoehn** [reply]
Great, thank you! So you mean it could be the same as my ssh username? At least this was I got when running `ps aux | egrep '(apache|httpd)'` -> `u10XXX+ 64669  0.0  0.0   3080   828 pts/0    SN+  14:54   0:00 grep -E (apache|httpd)`
